build.gradle.kts

```diff
@@ -1,6 +1,7 @@
 plugins {
   id("org.hypertrace.docker-plugin") version "0.7.1"
   id("org.hypertrace.docker-publish-plugin") version "0.7.1"
+  id("org.hypertrace.docker-java-application-plugin") version "0.7.1" apply false
```

Comment:
Any reason why this was added? Not needed here.

Reply:
copy pasta

```suggestion
```